EventFilter/HcalRawToDigi/src/HcalUnpacker.cc

```diff
@@ -588,8 +584,6 @@ void HcalUnpacker::unpackUTCA(const FEDRawData& raw, const HcalElectronicsMap& e
     int nps=(amc13->AMCId(iamc)>>12)&0xF;
     
     HcalUHTRData uhtr(amc13->AMCPayload(iamc),amc13->AMCSize(iamc));
-    //Check to make sure uMNio is not unpacked here
```

Comment:
Changes from this line onwards are unneeded and incorrect. Did you get the wrong version of the file?